patch/options.go

```diff
@@ -26,3 +27,19 @@ func fieldOptions(f *protogen.Field) *gopb.Options {
 func oneofOptions(o *protogen.Oneof) *gopb.Options {
 	return proto.GetExtension(o.Desc.Options(), gopb.E_Oneof).(*gopb.Options)
 }
+
+func getBoolExtension(pb proto.Message, extension protoreflect.ExtensionType, ifnotset bool) bool {
+	value := proto.GetExtension(pb, extension)
+	if v, ok := value.(bool); ok {
+		return v
+	}
+	return ifnotset
+}
+
+func fileEnumPrefixAllOption(f *protogen.File) bool {
+	return getBoolExtension(f.Proto.Options, gopb.E_EnumPrefixAll, false)
+}
+
+func fileFixIDFieldNamesOption(f *protogen.File) bool {
```

Comment:
While I agree that Id should be ID, it shouldn't be special cased.

My thought for this was to have a default, possibly extensible, list of initialisms like ID, HTTP, MIME, etc. Then couple this with message or file-level options to fix them automatically.

Reply:
Thank you for the quick review.
I improved a few things:

1. Global options for file. Example:
```
option (go.file) = {
    disable_enum_prefixes: true,
    fix_initialisms: true
 };
```
2. I added a function to check field names and a list of commonInitialisms. This code is from golang linter.

3. I added the ability to set custom initialisms list:
```
option (go.file) = {
    fix_initialisms: true,
    initialisms: ["ID", "WHATEVER"]    
 };
```
What do you think about these changes?